build.js

```diff
@@ -44,28 +44,18 @@ const markedOptions = {
 // in DEFAULT_LANG as a fallback to prevent any strings that aren't filled out
 // from appearing as blank.
 function i18nJSON (lang) {
-  var defaultJSON = require(`./locale/${DEFAULT_LANG}/site.json`)
-  var templateJSON = require(`./locale/${lang}/site.json`)
-  var finalJSON = JSON.parse(JSON.stringify(defaultJSON))
-  var merge = function (targetJSON, customJSON) {
-    Object.keys(customJSON).forEach(function (key) {
-      let value = customJSON[key]
-      if (typeof value === 'object') {
-        merge(targetJSON[key], value)
-      } else {
-        targetJSON[key] = value
-      }
-    })
-  }
-  merge(finalJSON, templateJSON)
-  return finalJSON
+  const defaultJSON = require(`./locale/${DEFAULT_LANG}/site.json`)
+  const templateJSON = require(`./locale/${lang}/site.json`)
+  const finalJSON = JSON.parse(JSON.stringify(defaultJSON))
```

Comment:
Yeah, definitely. We didn't need `JSON.parse(JSON.stringify(obj))` for deep copy.
I will modify it:)